### PR TITLE
updated visited item aria-label

### PR DESCRIPTION
**Pull request by kirsty-hames**
Default label set in hotgraphic.hbs includes item number and title. Visited label should include these too.

### CODE REVIEW

File: js/adapt-contrib-hotgraphic.js [hunk 1]
```diff
@@ -212,8 +212,12 @@ define(function(require) {
         setVisited: function(index) {
             var item = this.model.get('_items')[index];
             item._isVisited = true;
-            this.$('.hotgraphic-graphic-pin').eq(index).addClass('visited').attr('aria-label', "Item visited.");
+            this.$('.hotgraphic-graphic-pin').eq(index).addClass('visited');
             $.a11y_alert("visited");
+            var currentIndex = this.$('.hotgraphic-item.active').index();
+            var itemTitle = this.model.get('_items')[index].title;
+            var visitedLabel = Adapt.course.get('_globals')._accessibility._ariaLabels.visited;
```
Comment (moloko) [inline js/adapt-contrib-hotgraphic.js]: Line 219 might be better as 
```
var visitedLabel = this.model.get('_globals')._accessibility._ariaLabels.visited;
```
?

**Comment by moloko** [comment]
Right, so if I resize the component so that it changes to narrative style, then resize back, the 'visited' label is lost
I think this does also require some changes to the handlebars template to avoid this. I don't mind doing this as I've done something similar before for our hotgrid component

**Comment by oliverfoster** [comment]
+1 there are no checks in here for the existence of course._globals._accessiblity._ariaLabels, is that ok?

**Comment by brian-learningpool** [comment]
+1

**Comment by hrajotia** [comment]
+1

**Pull request merged**